redfish-core/include/utils/chassis_utils.hpp

```diff
@@ -178,27 +167,20 @@ inline void checkForAssemblyAssociations(
     using associationList =
         std::vector<std::tuple<std::string, std::string, std::string>>;
```

Comment:
It looks like - this associationList is also used in assembly.hpp.
Perhaps, can we put this into ./include/dbus_utility.hpp like

```
    using AssociationList  =
        std::vector<std::tuple<std::string, std::string, std::string>>;
```

And implement a `dbus::utility::getAssociationList()` - using `getProperty<AssociationList>`?